x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/notification/email/EmailService.java

```diff
@@ -108,11 +110,7 @@
     private final CryptoService cryptoService;
 
     public EmailService(Settings settings, @Nullable CryptoService cryptoService, ClusterSettings clusterSettings) {
-        super("email", clusterSettings, Arrays.asList(SETTING_DEFAULT_ACCOUNT, SETTING_PROFILE, SETTING_EMAIL_DEFAULTS, SETTING_SMTP_AUTH,
-                SETTING_SMTP_STARTTLS_ENABLE, SETTING_SMTP_STARTTLS_REQUIRED, SETTING_SMTP_HOST, SETTING_SMTP_PORT, SETTING_SMTP_USER,
-                SETTING_SMTP_PASSWORD, SETTING_SMTP_TIMEOUT, SETTING_SMTP_CONNECTION_TIMEOUT, SETTING_SMTP_WRITE_TIMEOUT,
-                SETTING_SMTP_SSL_TRUST_ADDRESS, SETTING_SMTP_LOCAL_ADDRESS, SETTING_SMTP_LOCAL_PORT, SETTING_SMTP_SEND_PARTIAL,
-                SETTING_SMTP_WAIT_ON_QUIT));
+        super("email", settings, clusterSettings, getClusterSettings());
```

Comment:
the naming might be a bit confusing: `clusterSettings` and then `getClusterSettings()` as variable names

Reply:
I have renamed `getClusterSettings` to `getDynamicSettings` . The fact that they are cluster settings is not relevant in this context (they are all node scope) what is relevant is that they are dynamic (Property.Dynamic).